compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -832,7 +832,12 @@ OMR::Z::CodeGenerator::CodeGenerator()
 bool
 OMR::Z::CodeGenerator::getSupportsBitPermute()
    {
-   return true;
+   if (TR::Compiler->target.is64Bit() || self()->use64BitRegsOn32Bit())
+      {
+      return true;
+      }
+      
+   return false;
```

Comment:
Why not just `return TR::Compiler->target.is64Bit() || self()->use64BitRegsOn32Bit()`?

Reply:
Done: 
e3cd762